libs/template/README.md

```diff
@@ -53,6 +53,21 @@ are several techniques that need to be known and considered.
 or  
 `yarn add @rx-angular/template`
 
+## Usage
+
+Import `TemplateModule` to your `AppModule`.
+
+```typescript
+import { TemplateModule } from '@rx-angular/template';
+
+@NgModule({
+  declarations: [...],
+  imports: [TemplateModule],
+})
+export class AppModule {}
+```
+
+
```

Comment:
```suggestion
## Usage

Import `TemplateModule` to your Module.

```typescript

import { TemplateModule } from '@rx-angular/template';

@NgModule({
  declarations: [...],
  imports: [TemplateModule],
})
export class MyModule {}
```

Alternatively, you can import `LetModule` or `PushModule` individually.

```typescript
import { LetModule, PushModule } from '@rx-angular/template';

@NgModule({
  declarations: [...],
  imports: [LetModule, PushModule],
})
export class MyModule {}
```

```

Reply:
@hoebbelsB smth like this?